etc/chart_schema.yaml

```diff
@@ -20,3 +21,17 @@ maintainer:
   name: str()
   email: str(required=False)
   url: str(required=False)
+---
+dependency:
+  name: str()
+  version: str()
+  repository: str()
+  condition: str(required=False)
+  tags: list(str(), required=False)
+  enabled: bool()
```

Comment:
This field is optional according to https://helm.sh/docs/topics/charts/#the-chartyaml-file

Reply:
@smlx Thanks. I've just updated that